doc/CONFIGURATION.md

```diff
@@ -90,6 +92,8 @@ See a [sample](./etc/pgmoneta.conf) configuration for running `pgmoneta` on `loc
 | tls_cert_file | | String | No | Certificate file for TLS. This file must be owned by either the user running pgmoneta or root. |
 | tls_key_file | | String | No | Private key file for TLS. This file must be owned by either the user running pgmoneta or root. Additionally permissions must be at least `0640` when owned by root or `0600` otherwise. |
 | tls_ca_file | | String | No | Certificate Authority (CA) file for TLS. This file must be owned by either the user running pgmoneta or root.  |
+| network_max_rate | 0 | Int | No | The number of bytes of tokens added every one second to limit the netowrk backup rate. 0 means no limit or uses global configuration.|
```

Comment:
Same - but `-1` is default

Reply:
Is there a reason for `-1` to be default? I think `0` works fine too.